pkg/config/user_config.go

```diff
@@ -456,6 +457,7 @@ func GetDefaultConfig() *UserConfig {
 			ShowListFooter:            true,
 			ShowCommandLog:            true,
 			ShowBottomLine:            true,
+			ShowPanelJumps:            false,
```

Comment:
I'm wondering if the default should be true. This is mostly for the benefit of new users, isn't it? Experienced users probably don't need it. But new users often don't know what configs exist or how to set them.

Reply:
I don't disagree, but I was afraid that setting it to `true` might annoy existing users who don't want the new label.

Reply:
Yes, but existing users are more likely to be able to figure out how to turn it off than new users are to figure out how to turn it on, or know that it even exists.

But also, I'm a big fan of doing things in a way that they don't need a config at all. If we add a feature that half of our users are annoyed by, we shouldn't add it, or rather try harder to add it in such a way that it doesn't annoy anybody. I feel that putting the number in front of the tab titles could be done unconditionally without a config.

Reply:
I agree: this makes sense to default to true